README.md

```diff
@@ -31,6 +31,10 @@ C++ implementation of [Apache Iceberg™](https://iceberg.apache.org/).
 - CMake 3.25 or higher
 - C++23 compliant compiler
 
+## Customizing Dependency URLs
+
+If you experience network issues when downloading dependencies, you can customize the download URLs using environment variables. See the available options documented in [`cmake_modules/IcebergThirdpartyToolchain.cmake`](cmake_modules/IcebergThirdpartyToolchain.cmake#L24-L40).
```

Comment:
IcebergThirdpartyToolchain.cmake can be changed, the link you put here is not stable. Why not directly put the quoted lines here?